apps/ai-content-generator/src/components/app/dialog/common-generator/output/output-text-panels/OutputTextPanels.tsx

```diff
@@ -22,10 +22,7 @@ const OutputTextPanels = (props: Props) => {
   const sdk = useSDK<DialogAppSDK>();
 
   const handleEntryApply = async () => {
-    const fieldKey = outputField.toLocaleLowerCase();
```

Comment:
This was making the `outputField` lowercase and therefore causing errors where the field wasn't recognized, so I removed it.